packages/replayio/src/utils/prompt/updatePrompt.ts

```diff
@@ -0,0 +1,14 @@
+import { readFromCache, writeToCache } from "../cache";
+import { promptHistoryPath } from "./config";
+import { PromptHistory } from "./types";
+
+export function updatePrompt({ id, metadata }: { id: string; metadata: any }) {
```

Comment:
maybe smth like `updatePromptCache`? I was a little bit confused when seeing the call to this function in isolation since it read as if we are updating the currently displayed prompt or something

Reply:
Sure